packages/block-template/templates/html/block-metadata.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "html-block",
+  "version": "0.0.0",
+  "description": "Block Protocol block template for an HTML element-based block",
+  "author": {
```

this is a `string` in BP (doesn't matter in our other templates because `create-block-app` will replace it with the user's git name - we could update CBA to update this file for HTML blocks in the same way)
```suggestion
  "author": "HASH"
```